include/aws/mqtt/client.h

```diff
@@ -239,8 +239,8 @@ int aws_mqtt_client_connection_connect(
     struct aws_mqtt_client_connection *connection,
     const struct aws_byte_cursor *host_name,
     uint16_t port,
-    struct aws_socket_options *socket_options,
-    struct aws_tls_connection_options *tls_options,
+    const struct aws_socket_options *socket_options,
+    const struct aws_tls_connection_options *tls_options,
```

Comment:
Mention in the connect() docs that this stuff is copied now

Reply:
Yes!